Makefile

```diff
@@ -205,7 +212,7 @@ deploy: manifests ## Deploy controller to the K8s cluster specified in ~/.kube/c
 		k8s.gcr.io/kustomize/kustomize:v${KUSTOMIZE_VERSION} edit set image controller=${MANAGER_IMG}
 	docker run --rm -v $(shell pwd)/config:/config \
 		k8s.gcr.io/kustomize/kustomize:v${KUSTOMIZE_VERSION} build \
-		/config/default | kubectl apply -f -
+		/config/default | kubectl apply --server-side -f -
```

Comment:
we should be okay to drop --server-side now?

Reply:
Whoops, I removed another but missed this one.

Done